packages/browser/src/core/analytics/index.ts

```diff
@@ -102,6 +102,7 @@ export interface InitOptions {
         aid?: RegExp
         uid?: RegExp
       }
+  highEntropyValues?: string[]
```

Comment:
`requestClientHintsHighEntropyValues` or something more descriptive that ties this to Client Hints?

Reply:
Is it actually necessary atm tho? I don't think this option was mentioned is the SDD. Seems like if you don't pass any hints specifically, it just gets everything it has -- seems OK for 99%, no?


... anyway, If we think we want to add it not or down the road, my vote is for "getHighEntropyValuesParams", since it's just the arguments you directly pass to https://developer.mozilla.org/en-US/docs/Web/API/NavigatorUAData/getHighEntropyValues


 (Also -- small things if we want to add it
1. A doc string in the interface would be useful.
2. Would want a unit test to confirm the public API behavior.)

Reply:
It's just typescript typings -- we can always expand on the list if it does change.